sdk/keyvault/azure-keyvault-keys/tests/test_crypto_client_async.py

```diff
@@ -878,3 +881,9 @@ async def test_decrypt_argument_validation():
     with pytest.raises(ValueError) as ex:
         await client.decrypt(EncryptionAlgorithm.rsa_oaep, b"...", authentication_tag=b"...")
     assert "authentication_tag" in str(ex.value)
+    with pytest.raises(ValueError) as ex:
+        await client.decrypt(EncryptionAlgorithm.a128_gcm, b"...", iv=b"...")
```

Comment:
Consider doing this exhaustively, i.e. for every algorithm.

Reply:
Good idea -- will do in a separate PR